app/assets/javascripts/controllers/dialog_user/dialog_user_controller.js

```diff
@@ -0,0 +1,25 @@
+ManageIQ.angular.app.controller('dialogUserController', ['API', 'dialogFieldRefreshService', 'serviceTemplateId', 'serviceTemplateCatalogId', function(API, dialogFieldRefreshService, serviceTemplateId, serviceTemplateCatalogId) {
+  var vm = this;
+
+  vm.$onInit = function() {
+    return new Promise((resolve) => {
+      resolve(API.get(`/api/service_templates/${serviceTemplateId}/service_dialogs?expand=resources&attributes=content`).then(init));
```

Comment:
Can't use ES6 in `app/assets/javascripts/`, you'd have to move this to `app/javascript/`.

Reply:
😮 this is goooooooooooooooood to know

Reply:
Yeah..
`app/assets/javascripts` => the old rails pipeline - sprockets, no JS compilation
`app/javascript` => the webpack way - yes to es6, yes to TS, yes to everything! :)